lib/src/models/result_movie.dart

```diff
@@ -55,8 +55,8 @@ class ResultMovie {
       posterPath: parsedJson['poster_path'] != null
           ? MovieStrings.imagesPath + parsedJson['poster_path']
           : MovieStrings.defaultImage,
-      originalLanguage:
-          parsedJson['original_language'] ?? MovieStrings.languageNotAssigned,
+      originalLanguage: parsedJson['original_language'].toUpperCase() ??
```

Comment:
Here is not necessary to do de toUpperCase(),you need to convert to upperCase when you are going to use it i mean that when you create MovieDetail( ... , originalLanguage: snapshot.data.results[index].originalLanguage.toUpperCase(), ... ) that is a correct implementation

Reply:
Fixed! Thank you